common/web/types/src/util/common-events.ts

```diff
@@ -0,0 +1,21 @@
+import { CompilerErrorNamespace, CompilerErrorSeverity, CompilerEvent } from './compiler-interfaces.js';
+import { ErrorObject } from "ajv";
+
+const m = (code: number, message: string) : CompilerEvent => { return { code, message } };
```

Comment:
This could be imported from `compiler-interfaces.js` thusly, and thereby share the definition:

```js
import { CompilerMessageSpec as m } from './compiler-interfaces.js';
```

Reply:
+1